### ISSUE TITLE
GitHub Actions support?

### ISSUE THREAD

**Issue by orj** [mergeability/mergeable#323]
As a user
I'd like to run mergeable robot as a GitHub Action workflow
So that I don't need to deploy and run a probot service

There appears to be an existing https://github.com/probot/actions-adapter that could be used.

**Comment by shine2lay** [comment]
@orj are you aware you can just install mergeable app that we hosted [here](https://github.com/apps/mergeable) ?

**Comment by orj** [comment]
@shine2lay I work for an org that will have issues with using a 3rd party app like that.  They will want to install Mergeable on their own infrastructure or use it as an Action.  They would be afraid that Mergeable will receive confidential information out of our source code/repos.

**Comment by jusx** [comment]
Hi @orj -- your use case is understandable until we have GH actions compatibility, would you consider self hosting?

**Comment by orj** [comment]
@jusx Yeah, that would be how we'll have to do it for the time being.  Hence my request for Github Actions support.

**Comment by jusx** [comment]
👍

**Comment by dgteixeira** [comment]
This Support would also include the possibility of triggering GitHub Actions through mergeable `Actions` block?
@shine2lay would my idea be a different feature request than this one?

**Comment by jusx** [comment]
> This Support would also include the possibility of triggering GitHub Actions through mergeable `Actions` block?
> 
> @shine2lay would my idea be a different feature request than this one?

@dgteixeira GA support at an action block level is a lot more granular and modular. I'd create a separate feature request and link it to this one.

**Comment by dgteixeira** [comment]
Hey @jusx , thanks for the reply!

I've opened a new issue here (#608), that details the requested feature of adding GA support within the `Actions` block.
